project-koku/koku
# add unique together constraint

This PR adds an additional `unique_together` constraint for the `reporting_ocpazuretags_summary`,  `reporting_ocpawstags_summary`, `reporting_ocpusagepodlabel_summary`, and `reporting_ocpstoragevolumelabel_summary` tables. The new constraints will resolve the follow error:

[https://sentry.io/organizations/project-koku/issues/1871212043/?referrer=slack](https://sentry.io/organizations/project-koku/issues/1871212043/?referrer=slack)

testing (with all services running in docker): 
1. Stop the worker: `docker-compose stop koku-worker`
2. `make create-test-customer`
3. `make load-test-customer-data`
4. Modify the generated OCP csv files:
a. within `testing/pvc_dir/insights_local/my-ocp-cluster-1` do find/replace on all files. Change `cost-management` -> `install-test`
b. within `testing/pvc_dir/insights_local/my-ocp-cluster-2` do find/replace on all files. Change `mobile` -> `banking`
5. Bring up the worker: `docker-compose up -d`
6. Ensure all data is ingest. No `ON CONFLICT` errors are raised.
7. Check `reporting_ocpazuretags_summary`, `reporting_ocpawstags_summary`, `reporting_ocpusagepodlabel_summary`, `reporting_ocpstoragevolumelabel_summary` db tables and ensure they are populated with data.
